src/styles/blocks/page.scss

```diff
@@ -0,0 +1,7 @@
+.page {
+  font-family: "Roboto", Arial, sans-serif;
+  font-weight: 400;
```

Comment:
Better to set the font family to the HTML tag, if you don't want to use a tag to set it, you can add class to the HTML tag